addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -46,6 +46,6 @@
     "protractor": "4.0.3",
     "ts-node": "1.2.1",
     "tslint": "3.13.0",
-    "typescript": "2.0.0"
+    "typescript": "^2.0.2"
```

Comment:
Is this really the right direction, having seen what just happened with ^2.0.0?  Do we really want to set things up so that everyone's CLI apps break when TS 2.0.3 comes out?